REAMDE

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
That's alright. I just add them out of pedantic convention, I suppose... although I suspect there was once a better reason that I got into such a habit! And you're wright: symlinks are a pain on Windows still.

Anyway, I'll leave the rest to you then?